hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/resources/hdfs-rbf-default.xml

```diff
@@ -723,6 +723,14 @@
     </description>
   </property>
 
+  <property>
+    <name>dfs.federation.router.fairness.acquire.timeout</name>
+    <value>1s</value>
```

Comment:
Is this configuration confirmed to be like this?

I feel the following is correct
```
<value>1</value>
```

Reply:
I see, your configuration is accurate.